Core/Services/StitchCoreRemoteMongoDBService/Tests/StitchCoreRemoteMongoDBServiceTests/Sync/DocumentVersionInfoUnitTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+import MongoMobile
+import MongoSwift
+@testable import StitchCoreRemoteMongoDBService
+
+class DocumentVersionInfoUnitTests: XCTestCase {
+    func testVersionDocument() {
```

Comment:
[minor] also add `testGetNextVersion` from the Android unit tests

Reply:
I am technically testing the next version here.